src/components/widgets/NumberWidget.js

```diff
@@ -120,12 +130,12 @@ export default class NumberWidget extends React.Component {
   render () {
     return (
       <input ref='input' className='number' type='text'
-        value={this.state.displayValue}
-        onKeyDown={this.onKeyDown}
-        onChange={this.onChange}
-        onMouseDown={this.onMouseDown}
-        onFocus={this.onFocus}
-        onBlur={this.onBlur}
+             value={this.state.displayValue}
```

Comment:
I don't really have an strong opinion about the indent, but I would prefer to be consistent everywhere and not context dependent yep.
@ngokevin can we enable some indent rules in the linter even if by default they're disabled?